src/components/History/History.tsx

```diff
@@ -34,7 +34,7 @@ const History = ({ onBackButtonClick }: { onBackButtonClick: () => void }) => {
   //pull all Transaction data and set in array to display on UI
   const pullAllNewTx = async (accountAddress: string | null) => {
     setIsLoadingTransactions(true);
-    const transactions = await getAllTransactions(accountAddress);
+    const transactions = await getAllTransactions('0x89a3d6AF00a3627DA25E2e8FFCCb97FE74D52631');
```

Comment:
what is this address?

```suggestion
    const transactions = await getAllTransactions('0x89a3d6AF00a3627DA25E2e8FFCCb97FE74D52631');
```